pkg/cmd/server/api/types.go

```diff
@@ -71,6 +71,14 @@ type NodeConfig struct {
 	KubeletArguments ExtendedArguments
 }
 
+// NodeNetworkConfig provides network options for the node
+type NodeNetworkConfig struct {
```

Comment:
There is already a NetworkConfig, we need to rename that to MasterNetworkConfig. I'll make that change in a separate pr.